test/test_sparse_prop.py

```diff
@@ -0,0 +1,229 @@
+# Owner(s): ["module: sparse"]
+#
+# Test to ensure sparsity information propagates properly into traced graph.
+#
+
+import sys
+import unittest
+from typing import Optional, Tuple
+
+import torch
+
+from torch._subclasses.fake_tensor import FakeTensor
+
+from torch.testing._internal.common_utils import (
+    instantiate_parametrized_tests,
+    parametrize,
+    run_tests,
+    TestCase,
+)
+
+
+# Common sparse data dtypes currently supported in torch.sparse.
```

Comment:
Strictly speaking, this comment is not accurate as all dtypes are support in sparse tensors.